core/src/main/java/com/netflix/iceberg/ReplaceFiles.java

```diff
@@ -16,21 +16,113 @@
 
 package com.netflix.iceberg;
 
-import java.util.List;
-import java.util.Set;
+import com.netflix.iceberg.exceptions.CommitFailedException;
+import com.netflix.iceberg.exceptions.RuntimeIOException;
+import com.netflix.iceberg.exceptions.ValidationException;
+import com.netflix.iceberg.io.OutputFile;
+
+import java.io.IOException;
+import java.util.*;
```

Comment:
Style: please don't use globbed imports because you can't search for class usage

Reply:
Fixed.